src/middleware/index.ts

```diff
@@ -7,7 +7,7 @@ import setupRateLimiter from './rateLimiterSetup';
 import { ExpressMiddlewareConfig, ExpressMiddlewareSet } from '../@types/expressMiddleware';
 import { RateLimiterResponse } from '../@types/rateLimit';
 import { connect } from '../utils/redis';
-import ComplexityAnalysis from '../analysis/ASTParser';
+import ASTParser from '../analysis/ASTParser';
```

Comment:
What was the reason for changing this back? Not against it but if we're making this change here for clarity then maybe we come up with a new name for the class

Reply:
Your documentation had the `ComplexityAnalysis` class as type `ASTParser`. It is still exposed as `ComplexityAnalysis` but with `type ASTParser`

Reply:
When you click `ComplexityAnalysis` to see the definition it brings you to `ASTParser`. What do you think is more readable?